### DISCUSSION TITLE
Make it possible to trigger only for non-draft PRs

### DISCUSSION THREAD

**Discussion by norman-zon** [community/community#102838] [Actions]
### Select Topic Area

Product Feedback

### Body

We have a lot of CI jobs, we only want to run for PRs, that are not drafts. Developers open PRs as drafts and just when they become ready to review, workflows should be triggered.

From my understanding this is not possible right now as the `ready_for_review` type only is triggered by the transition from draft to non-draft.

Checking for `github.event.pull_request.draft` inside the workflow comes to late. We don't want to trigger at all, so we don't assign a runner to determine there's nothing to do and exit.

Maybe `AND`ing triggers would be an option? Like `synchronize && ready_for_review`?

**Comment by Marcono1234** [comment]
Related to #25722, but so far the answers there are using `github.event.pull_request.draft`, which as you mention does not work for your use case.
